### PR TITLE
Add base images to fast-forward

### PR DESCRIPTION
**Pull request by mqhackett**
Related Issue: open-cluster-management/backlog#10511

Description:
 - Add base image info to fast-forward job in config.

### CODE REVIEW

File: ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml [hunk 1]
```diff
@@ -55,6 +55,11 @@ tests:
   - mount_path: /etc/sonarcloud/
     name: acm-sonarcloud-token
 - as: fast-forward
+  base_images:
```
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml:58]: The repo-brancher entry needs to be part of the base_images stanza that starts on line 1.

**Review by jlpadilla — CHANGES_REQUESTED** [review]
@mqhackett We need to update all repos, not just the collector.

**Comment by jlpadilla** [comment]
@mqhackett search-api is missing.

**Review by jlpadilla — APPROVED** [review]

**Pull request merged**